### PR TITLE
Update description of `options.type` within connection resource

### PR DESCRIPTION
**Pull request by kopernic-pl**
Description added per what's in Auth0 frontend next to this parameter.

### 🔧 Changes

Description of parameter added.

### 📚 References

image
As on attached image.

### 🔬 Testing

build run locally, passing

### 📝 Checklist

- [ N/A] All new/changed/fixed functionality is covered by tests (or N/A)
- [ N/A] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

**Review by sergiught — CHANGES_REQUESTED** [review]
Hey @kopernic-pl 👋🏻 thanks for the contribution! Would you mind running a quick made docs and commit any changed files as well? 🙏🏻

File: internal/auth0/connection/schema.go [hunk 1]
```diff
@@ -569,7 +569,9 @@ var resourceSchemaV0 = map[string]*schema.Schema{
 				"type": {
 					Type:        schema.TypeString,
 					Optional:    true,
-					Description: "Value can be `back_channel` or `front_channel`.",
+					Description: "Value can be `back_channel` or `front_channel`. " +
+						"Front Channel will use OIDC protocol with response_mode=form_post and response_type=id_token. " +
```
Comment (sergiught) [inline internal/auth0/connection/schema.go:573]:
```suggestion
						"Front Channel will use OIDC protocol with `response_mode=form_post` and `response_type=id_token`. " +
```

File: internal/auth0/connection/schema.go [hunk 2]
```diff
@@ -569,7 +569,9 @@ var resourceSchemaV0 = map[string]*schema.Schema{
...
+ 						"Back Channel will use response_type=code.",
```
Comment (sergiught) [inline internal/auth0/connection/schema.go:574]:
```suggestion
 						"Back Channel will use `response_type=code`.",
```

**Comment by kopernic-pl** [comment]
`make docs` change committed. Re-review requested.

**Review by sergiught — APPROVED** [review]
Thanks a lot for the contribution @kopernic-pl 💯

**Pull request merged**